bootblocks/boot_minix.c

```diff
@@ -74,18 +79,22 @@ static void load_file ();
 void load_prog()
 {
 	// Avoid reuse of an old copy of /bootopts in memory if we're rebooting w/ no /bootopts */
-	int __far *optseg = _MK_FP(OPTSEG, 0);	/* Expensive, is there a better way? */
-	*optseg = i_boot = i_now = 0;
+	//int __far *optseg = _MK_FP(OPTSEG, 0);	/* Expensive, is there a better way? */
+	i_boot = i_now = 0;
+	//*optseg = i_boot = i_now = 0;
```

Comment:
Since you've commented out the previous fix to zero out the first bytes of the OPTSEG, how is it that this still works when /bootopts is deleted and then the system rebooted?